pyproject.toml

```diff
@@ -27,6 +27,9 @@ kgcl-rdflib = "^0.3.0"
 pystow = "^0.4.4"
 class-resolver = "^0.3.10"
 ontoportal-client = "0.0.3"
+curies = "^0.1.5"
```

Comment:
[prefixmaps already has a curies dependency](https://github.com/linkml/prefixmaps/blob/bdd770a5f65c6fc16a229d9632771db659b30a54/pyproject.toml#L16). Isn't this redundant?

Reply:
no it should definitely be explicit since this package invokes `curies` directly

Reply:
agreed... and I don't think prefixmaps needs that dependency. It's literally just a dumb named prefix->namespace mapping cache